src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtMediaTypePriceGranularity.java

```diff
@@ -26,5 +27,6 @@
      * Defines the contract for bidrequest.ext.prebid.targeting.mediatypepricegranularity.native
      */
     @JsonProperty("native")
+    @Getter(onMethod = @__({@JsonProperty("native")}))
```

Comment:
Is it possible to add unit test to `org.prebid.server.handler.openrtb2.AuctionHandler` to cover this case?